Makefile

```
@@ -125,6 +139,15 @@ toc-verify:
 vet: ## Run go vet against code.
 	$(GO_CMD) vet ./...
 
+## Create Kind Cluster for E2E
+.PHONY: kind-create
+kind-create:
+	kind create cluster --image=$(E2E_KIND_VERSION) --name=$(KIND_CLUSTER_NAME)
```

```suggestion
	$(KIND) create cluster --image=$(E2E_KIND_VERSION) --name=$(KIND_CLUSTER_NAME)
```

Same for other calls